.travis.yml

```diff
@@ -131,8 +131,10 @@ script:
   # affect any downstream functionality but causes the build to fail
   # spuriously. The echo-backtick workaround gets around this error,
   # which should be investigated further in the future.
-  - if [[ $TEST_TARGET == 'doctest' ]]; then
+  - >
```

Comment:
What does this add that we didn't have before?

Reply:
Tells yaml to go into multi-line mode, and prevents travis from going crazy from the newly added pipe line. Without it, travis can't even read the config file.

Reply:
Previously, everything was concatenated into one line, so theoretically, you could also drop the semicolons on every line here as well.

Reply:
Good to know. I'm tempted to do this once a few of the changes to the travis.yml have been merged.